docs: clarify container label usage

EDIflyer opened the pull request:
As discussed in #1318 - just to aid clarity for new users and also make docker-compose element clear.

<!--

Thank you for contributing to the watchtower project! 🙏

We truly appreciate all the contributions we get from the community.

To make your PR experience as smooth as possible, make sure that you
include the following in your PR:

- What your PR contributes
- Which issues it solves (preferrably using auto closing instructions like "closes #123".
- Tests that verify the code your contributing
- Updates to the documentation

Thank you again! ✨

-->

EDIflyer committed:
clarify documentation around setting label

As discussed in #1318 - just to aid clarity for new users and also make docker-compose element clear.

EDIflyer committed:
changed to tabbed version for mkdocs

piksel commented on docs/container-selection.md:
````diff
@@ -7,29 +7,57 @@ There are two options:
+        labels:
+          - "com.centurylinklabs.watchtower.enable=false"
+    ```
+
````
Ehm.... 😅

>  For clarity this should be set **on the container(s)** you wish to be ignored, this is not set on watchtower.

EDIflyer replied:
Haha oops yeah good point 😂 I was trying to make my example really generic and stupidly picked watchtower as an example container 🤦🤦 I'll change to nginx!

EDIflyer replied:
OK now changed to `someimage` as per other examples - thanks for spotting!

EDIflyer committed:
changed example container

after stupidly using watchtower as the docker-compose example (doh!) have now updated to `someimage`, in-keeping with the other examples

piksel commented on docs/container-selection.md:
```diff
@@ -7,29 +7,67 @@ There are two options:
+        entrypoint:
+          - somepoint
+        environment:
+          - PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
```suggestion
      someimage:
        container_name: someimage
```
I would suggest leaving out all the stuff that is not relevant here, especially since they have "real" values. Setting the `PATH` to `/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin` is not relevant to this feature, so it would be up to the user reading the docs to guess what matters and what doesn't.

piksel commented on docs/container-selection.md:
```diff
@@ -7,29 +7,67 @@ There are two options:
+        entrypoint:
+          - somepoint
+        environment:
+          - PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
```suggestion
        container_name: someimage
```

Same as below. Only include the parts relevant for the feature.

piksel commented on docs/container-selection.md:
````diff
@@ -7,29 +7,67 @@ There are two options:
+    ```docker
+    LABEL com.centurylinklabs.watchtower.enable="false"
+    ```
+=== "dockerrun"
````
```suggestion
=== "docker run"
```

piksel commented on docs/container-selection.md:
````diff
@@ -7,29 +7,67 @@ There are two options:
+    ```docker
+    LABEL com.centurylinklabs.watchtower.enable="true"
+    ```
+=== "dockerrun"
````
```suggestion
=== "docker run"
```

piksel reviewed (changes requested):
Looks good overall, but I think the examples should be reduced a bit to make it clearer what is actually required.

EDIflyer committed:
Update docs/container-selection.md

Co-authored-by: nils måsén <nils@piksel.se>

EDIflyer committed:
Update docs/container-selection.md

Co-authored-by: nils måsén <nils@piksel.se>

EDIflyer committed:
Update docs/container-selection.md

Co-authored-by: nils måsén <nils@piksel.se>

EDIflyer committed:
Update docs/container-selection.md

Co-authored-by: nils måsén <nils@piksel.se>
